src/algebra/order/to_interval_mod.lean

```diff
@@ -457,6 +457,67 @@ begin
     rw [hz, to_Ioc_mod_zsmul_add] }
 end
 
+section Ico_Ioc
+
+variables (a : α) {b : α} (hb : 0 < b)
+
+lemma tfae_to_Ico_eq_to_Ioc (x : α) :
```

Comment:
can you add a docstring for this lemma, explaining what this is about?

Reply:
Added.